spv/envelope.go

```diff
@@ -42,39 +41,22 @@ func (e *Envelope) ParentTx(txID string) (*bt.Tx, error) {
 // Bytes takes an spvEnvelope struct and returns the serialised binary format.
 func (e *Envelope) Bytes() ([]byte, error) {
```

Comment:
this still says envelope here

Reply:
Leaving because most test data is written in this format.